1-ros_2_introduction/README.md

```diff
@@ -104,8 +104,6 @@ Open a new terminal inside the Docker container and run the following commands (
     ```
    <img src="images/ros2_topic_info.png" alt="ROS 2 topic info" width="600">
 
-We see that there is one publisher for the `/scan` topic, which is the driver node of the simulated lidar on Andino. What node is then subscribing to this topic?
```

Comment:
Why did you take this question out?

Reply:
Leaving out in this batch, as the question is not answered. We can add it later with an example how to get this information.